vllm/worker/model_runner.py

```diff
@@ -910,11 +1000,24 @@ def capture_model(self, kv_caches: List[torch.Tensor]) -> None:
                     self.set_active_loras(set(), lora_mapping)
 
                 graph_runner = CUDAGraphRunner(self.model)
+
+                model_inputs = {
```

Comment:
This is to generalize for models that have different input signature than transformers (like Medusa).